test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetListPackageTests.cs

```diff
@@ -84,6 +84,94 @@ public async Task DotnetListPackage_NoRestore_Fail()
             }
         }
 
+        [PlatformFact(Platform.Windows)]
```

Comment:
Should we also add a test for a global reference?

Reply:
Talked with Jeff about GlobalReferences and the feature its implemented, I'll add tests for that

Reply:
I'm signing off now assuming these tests will be added before merging.